lib/shared/sdk/usbboot/index.js

```diff
@@ -251,14 +261,31 @@ class USBBootAdapter extends EventEmitter {
     usb.listDevices().filter(isUsbBootCapableUSBDevice).map((device) => {
     /* eslint-enable lodash/prefer-lodash-method */
 
+      const description = COMPUTE_MODULE_DESCRIPTIONS[device.deviceDescriptor.idProduct] || 'Compute Module'
```

Comment:
Should this maybe default to something more generic, like "USBBoot Device"? If that makes sense, that is

Reply:
Nope, it's only the Raspberry Pi Compute Modules that implement this (proprietary) USB-boot protocol that we're dealing with here.